client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -174,38 +170,52 @@ h2.action-panel__title {
 }
 
 .email-provider-stacked-card__title-price-badge {
-	width: $width-right-panel-card;
+	width: 100%;
+	margin-left: 10px;
+	@include break-small {
+		width: $width-right-panel-card;
+	}
 }
 
 .email-provider-stacked-card__provider-form-and-right-panel {
 	display: none;
 }
 
+.email-providers-stacked-comparison__main.main .card.action-panel.promo-card.email-providers-stacked-comparison__provider-card .action-panel__figure {
+	margin-right: unset;
+	@include break-mobile {
+		margin-right: 24px;
+	}
+}
+
+.email-provider-stacked-card__provider-expand-cta {
+	width: 110px;
```

Comment:
This feels like a very bad move for i18n -- we have no idea how long the string in the button is going to be.